ext/node/polyfills/internal_binding/tcp_wrap.ts

```diff
@@ -300,7 +300,9 @@ export class TCP extends ConnectionWrap {
    * @return An error status code.
    */
   setNoDelay(noDelay: boolean): number {
-    this[kStreamBaseField].setNoDelay(noDelay);
+    if ("setNoDelay" in this[kStreamBaseField]) {
```

Comment:
From [this issue](https://github.com/denoland/deno/issues/26363), it seems that a null check might be needed here.